src/components/formattedNumber/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+/**
+ *
+ * @param {*} num - it is a number that we want to formate it as formatted number
+ * @return {string} - formatted version of input number
+ */
+const formateNumber = (num) => {
+  num = parseFloat(num);
+  const sign = num < 0 ? '-' : '';
```

Comment:
Not all branches of ternary operators are covered with unit tests in this file. The report says `58.33% Branches 7/12`